### ISSUE TITLE
Inconsistent lab titles

### ISSUE THREAD

**Issue by agodfrind** [oracle/learning-library#1311]
On  https://oracle.github.io/learning-library/data-management-library/autonomous-database/shared/adb-quickstart-workshop/ The list of labs on the right side (“Contents”) does not match the list on the left side …

**Comment by anooshapilli** [comment]
![image](https://user-images.githubusercontent.com/63126232/81584888-5c7d4800-9381-11ea-8096-4b963503e60d.png)

Hi, From what I understand, I think you are comparing the list on the left side with the contents on the right as highlighted in the above image. 

If so, the answer is No. They **do not match** because contents on the right side are the **titles that navigate to the labs in any given workshop**  While the list on the left is the **topics covered under each specific lab**.

Say, the user is working on adb-quickstart-workshop, the first thing to do is to select a Lab from the contents on the right side. Once a lab is selected, e.g. *Introduction to ADB on Shared Infrastructure*  user can scroll through the sections of that Lab or can jump into a specific section e.g. *Lab Breakdown* by clicking on any item from the list on the left. 

I hope this clarifies your question.

**Comment by agodfrind** [comment]
When I looked at it last week, it was very different … I did not include a screen capture, but it had  “lab n” mentioned on both sides and that was confusing. Now I see that we moved the lab descriptions to the “Lab Breakdown” left-side tab. And the labs on that tab now match the ones on the right. That works better

Is that workshop final now ?
--


Albert Godfrind | Geospatial and Graph technologies | Tel: +33 1 57 60 86 16 | Mobile: +33 6 09 97 27 23 | Twitter: @agodfrin
Oracle Server Technologies
Centre Régus - Nova Antipolis | 291 Rue Albert Caquot | 06550 Sophia Antipolis | France
